compiler/optimizer/LocalDeadStoreElimination.cpp

```diff
@@ -754,7 +754,7 @@ void TR::LocalDeadStoreElimination::adjustStoresInfo(TR::Node *node, TR_BitVecto
    // being stored into as unused so that any dead store encountered above
    // can be removed if it does not feed a use.
    //
-   if (node->getOpCode().isStore())
+   if (node->getOpCode().isStore() && !node->chkStoredValueIsIrrelevant())
```

Comment:
Should this be checking !node->storedValueIsIrrelevant() instead ?

Reply:
To make it consistent with what every other place that checks for irrelevant stores is using...

Reply:
This has been updated to use `storedValueIsIrrelevant`, requiring the explicit check of `isAutoOrParm` as well.